components/org.wso2.carbon.identity.application.authenticator.live/src/main/java/org/wso2/carbon/identity/application/authenticator/live/WindowsLiveOAuth2AuthenticatorConstants.java

```diff
@@ -33,8 +35,19 @@
     public static final String GIVEN_NAME_CLAIM_URI = "http://wso2.org/claims/givenname";
     public static final String LAST_NAME_CLAIM_URI = "http://wso2.org/claims/lastname";
     public static final String GENDER_CLAIM_URI = "http://wso2.org/claims/gender";
-    public static final String LOCALITY_CLAIM_URI = "http://wso2.org/claims/locality";
+    public static final String LOCALITY_CLAIM_URI = getClaimUriLocale();
 
     private WindowsLiveOAuth2AuthenticatorConstants() {
     }
+
+    private static String getClaimUriLocale() {
```

Comment:
Will we be using the same logic in multiple places?

Will there be any issue in introducing a common place to get this logic and reusing it as oppose to implementing it in multiple places?

Reply:
Fixed in commit 4734163c36579cd69deab666ff3bed691a1dd8ca